src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,415 @@
+import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
+import { customElement, html, property } from 'lit-element';
+import { Providers } from '../../Providers';
+import { ProviderState } from '../../providers/IProvider';
+import { PersonCardInteraction } from '../mgt-person/mgt-person';
+import { MgtTemplatedComponent } from '../templatedComponent';
+import { styles } from './mgt-person-card-css';
+
+/**
+ * Web Component used to show detailed data for a person in the
+ * Microsoft Graph
+ *
+ * @export
+ * @class MgtPersonCard
+ * @extends {MgtTemplatedComponent}
+ */
+@customElement('mgt-person-card')
+export class MgtPersonCard extends MgtTemplatedComponent {
+  /**
+   * Array of styles to apply to the element. The styles should be defined
+   * using the `css` tag function.
+   */
+  static get styles() {
+    return styles;
+  }
+
+  /**
+   * Set the person details to render
+   *
+   * @type {(MicrosoftGraph.User
+   *     | MicrosoftGraph.Person
+   *     | MicrosoftGraph.Contact)}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'person-details',
+    type: Object
+  })
+  public personDetails: MicrosoftGraph.User | MicrosoftGraph.Person | MicrosoftGraph.Contact;
+  /**
+   * Set the image of the person
+   * Set to '@' to look up image from the graph
+   *
+   * @type {string}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'person-image',
+    type: String
+  })
+  public personImage: string;
+
+  /**
+   * Renders additional details section on initial show, false or true
+   *
+   * @type {string}
+   * @memberof MgtPersonCard
+   */
+  @property({
+    attribute: 'showadditionaldetails',
```

Comment:
this should be null and also private as it's only for internal use

Reply:
My thoughts on making this public were, if the user wants to define a person-card which automatically shows additional details (the user doesn't have to click the arrow to show the details).

Reply:
I'd rather have this as private now and make it public later if there is demand. It's much easier making something public vs making something private later